modules/msk/mutect2/main.nf

```diff
@@ -0,0 +1,49 @@
+process MUTECT2 {
```

Comment:
I think you're missing a `tag.yml` and `main.nf.test.snap`. You might also need to double check some of the other parts of the testing.